test/unit/org/apache/cassandra/db/marshal/AbstractTypeTest.java

```diff
@@ -460,6 +460,25 @@ public void nested()
         });
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void nestedDuration()
+    {
+        qt().forAll(AbstractTypeGenerators.builder().withoutTypeKinds(COUNTER).withPrimitives(DurationType.instance).build()).checkAssert(type -> {
+            assertThat(type.referencesDuration()).isTrue();
+
+            assertThat(ReversedType.getInstance(type).referencesDuration()).isTrue();
+        });
```

Comment:
(cosmetic) nit: similarly for other new test cases added in this patch, break the long lines into multiple-line statement. 

```suggestion
        qt()
        .forAll(AbstractTypeGenerators.builder()
                                      .withoutTypeKinds(COUNTER)
                                      .withPrimitives(DurationType.instance)
                                      .build())
        .checkAssert(type -> {
            assertThat(type.referencesDuration()).isTrue();
            assertThat(ReversedType.getInstance(type).referencesDuration()).isTrue();
        });
```